http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/service/LeonardoService.scala

```diff
@@ -529,29 +575,9 @@ class LeonardoService(protected val dataprocConfig: DataprocConfig,
       //if you've got to here you at least have GetClusterDetails permissions so a 403 is appropriate if you can't actually stop it
       _ <- checkClusterPermission(userInfo, StopStartCluster, cluster, throw403 = true)
 
-      _ <- internalStopCluster(cluster)
+      _ <- clusterHelper.stopCluster(cluster)
     } yield ()
 
-  def internalStopCluster(cluster: Cluster): IO[Unit] =
```

Comment:
I moved a lot of functions to the clusterHelper as they are needed in backLeo now too.